docs/topics/http/views.txt

```diff
@@ -166,3 +166,40 @@ The :func:`~django.views.defaults.bad_request` view is overridden by
 
     Use the :setting:`CSRF_FAILURE_VIEW` setting to override the CSRF error
     view.
+
+Testing custom error views
+--------------------------
+
+To test the response of your custom error handler, you should raise an
+exception in the view function. Here is an example::
```

Comment:
"the view function" - the view we're adding is for tests only, so perhaps call it "a test view". Views are always callables so no need to name it a 'function'.